### ISSUE TITLE
non carica tutti i libri

### ISSUE THREAD

**Issue by sefirothmorpheus** [Tecnologica-Mente/HUB_Scuola_DRM_Remover#4]
ciao
come mai non carica tutti i libri presenti nella mia libreria?

![hub](https://github.com/Tecnologica-Mente/HUB_Scuola_DRM_Remover/assets/156643499/c0e7ae20-8760-4555-a3c8-43319c9e3d3d)

dovrebbero esserci 154 libri nel mio profilo (non 110, in quanto alcuni libri hanno + volumi), ma con questo tool ne vedo solo 26.

**Comment by Tecnologica-Mente** [comment]
Ciao, a volte succede ma non so dirti il motivo.
Prova con lo script PDFGrabber_Launcher che è più aggiornato e funziona su diverse piattaforme.

**Comment by Pigi63** [comment]
Ciao. Posso chiederti se il programma drm hub scuola funziona?

**Comment by sefirothmorpheus** [comment]
> Ciao. Posso chiederti se il programma drm hub scuola funziona?

no, non funziona, non è stato aggiornato e penso l'autore non voglia aggiornarlo

**Comment by Tecnologica-Mente** [comment]
Ciao. L'unico script aggiornato per scaricare libri dalla maggior parte delle piattaforme online è [PDFGrabber_Launcher](https://github.com/Tecnologica-Mente/PDFGrabber_Launcher) che si appoggia a PDFGrabber di FelixFrog. Puoi provare utilizzando quello. Se non dovesse funzionare penso che per il momento sia tutto bloccato. Purtroppo le case editrici stanno correndo ai ripari e hanno fatto anche chiudere un canale YouTube dove erano presenti svariati video con tutte le procedure per scaricare...

**Comment by mircot80** [comment]
Dove posso scaricare PDFGrabber di FelixFrog ?

**Comment by Tecnologica-Mente** [comment]
Ciao. Penso che FelixFrog abbia avuto qualche problema con le case editrici dei libri... Ha reso il suo account privato, non risulta più neanche contattabile... L'unico script da utilizzare per ora sembra questo [https://github.com/RealRoti/pdfgrabber-advanced](url), ma non verrà più aggiornato...

**Comment by mircot80** [comment]
> Ciao. Penso che FelixFrog abbia avuto qualche problema con le case editrici dei libri... Ha reso il suo account privato, non risulta più neanche contattabile... L'unico script da utilizzare per ora sembra questo [https://github.com/RealRoti/pdfgrabber-advanced](url), ma non verrà più aggiornato...

Perfetto funziona, grazie.

**Comment by ludep93** [comment]
> Ciao. Penso che FelixFrog abbia avuto qualche problema con le case editrici dei libri... Ha reso il suo account privato, non risulta più neanche contattabile... L'unico script da utilizzare per ora sembra questo [https://github.com/RealRoti/pdfgrabber-advanced](url), ma non verrà più aggiornato...

Ciao, funziona ancora il link che hai inviato? Clicco ma mi esce una pagina vuota di gitbhub.

Grazie.

**Comment by Tecnologica-Mente** [comment]
Quale, quello di PDFGrabber-Advanced? Si, funziona, è stato archiviato e impostato di sola lettura. Come riporta nella home page: "The original project has been discontinued. For this reason, no more updates will be relased. Last update from source: 11 dec 2024", cioè : "Il progetto originale è stato interrotto. Per questo motivo, non verranno rilasciati ulteriori aggiornamenti. Ultimo aggiornamento dalla fonte: 11 dicembre 2024".

**Comment by ludep93** [comment]
> Quale, quello di PDFGrabber-Advanced? Si, funziona, è stato archiviato e impostato di sola lettura. Come riporta nella home page: "The original project has been discontinued. For this reason, no more updates will be relased. Last update from source: 11 dec 2024", cioè : "Il progetto originale è stato interrotto. Per questo motivo, non verranno rilasciati ulteriori aggiornamenti. Ultimo aggiornamento dalla fonte: 11 dicembre 2024".

Scusami, forse sbaglio io, o non riesco a trovarlo, ma quando clicco esce la pagina di github con scritto no results, try adjusting your search filter.

**Comment by Tecnologica-Mente** [comment]
Ho verificato. Ti da quel risultato se clicchi direttamente sul link oppure se apri il collegamento in una nuova scheda o in una nuova finestra. Seleziona invece il link, copialo e incollalo direttamente nella barra degli indirizzi del tuo browser, vedrai che troverai il file che cerchi. Non so perché, ma nel primo caso i collegamenti vengono modificati...
